cfme/tests/infrastructure/test_snapshot.py

```diff
@@ -109,13 +112,22 @@ def test_snapshot_crud(small_test_vm, provider):
         casecomponent: Infra
         initialEstimate: 1/6h
     """
+    result = LogValidator(
+        "/var/www/miq/vmdb/log/evm.log",
+        failure_patterns=[r".*ERROR.*"],
```

Comment:
Q: Is there any more specific `ERROR` message we could search for here? We wouldn't want this test to fail due to an error in an unrelated part of the appliance.

Reply:
I don't know about more specific errors that could happen, I followed verification steps https://bugzilla.redhat.com/show_bug.cgi?id=1571291#c13 .. and there is written that there should be no errors in evm.log.. So I guess multiple things could go wrong

Reply:
@psimovec I asked @lina-nikiforova the same question on one of her PR's and it seems like the same thing applies here. https://github.com/ManageIQ/integration_tests/pull/9375#discussion_r327011168